test/test_tomopy/test_misc/test_corr.py

```diff
@@ -112,31 +112,57 @@ def test_median_filter_nonfinite(self):
                     size=3,
                     callback=None,
                 )
-    
+
     def test_median_filter3d(self):
-        A = np.arange(4*5*6).reshape(4,5,6)
+        A = np.arange(4 * 5 * 6).reshape(4, 5, 6)
         assert_equal(
-            scipy.ndimage.median_filter(np.float32(A), size=3), 
-            median_filter3d(np.float32(A), size=3))
+            scipy.ndimage.median_filter(np.float32(A), size=3),
+            median_filter3d(np.float32(A), size=3),
+        )
 
     def test_remove_outlier3d(self):
-        A = np.arange(4*5*6).reshape(4,5,6)
-        A[2,2,2] = 1000.0 # introduce an outlier
-        A_dezinged = remove_outlier3d(np.float32(A), dif = 500, size=3)
-        A[2,2,2] = 75 # substituted value by dezinger
+        A = np.arange(4 * 5 * 6).reshape(4, 5, 6)
+        A[2, 2, 2] = 1000.0  # introduce an outlier
+        A_dezinged = remove_outlier3d(np.float32(A), dif=500, size=3)
+        A[2, 2, 2] = 75  # substituted value by dezinger
         assert_equal(np.float32(A), A_dezinged)
-        
+
     def test_remove_neg(self):
         assert_allclose(remove_neg([-2, -1, 0, 1, 2]), [0, 0, 0, 1, 2])
 
     def test_remove_nan(self):
-        assert_allclose(remove_nan([np.nan, 1.5, 2, np.nan, 1]),
-                        [0, 1.5, 2, 0, 1])
+        assert_allclose(remove_nan([np.nan, 1.5, 2, np.nan, 1]), [0, 1.5, 2, 0, 1])
 
     def test_remove_outlier(self):
-        proj = read_file('proj.npy')
+        proj = read_file("proj.npy")
         proj[8][4][6] = 20
-        assert_allclose(remove_outlier(proj, dif=10), read_file('proj.npy'))
+        assert_allclose(remove_outlier(proj, dif=10), read_file("proj.npy"))
 
     def test_circ_mask(self):
-        loop_dim(circ_mask, read_file('obj.npy'))
+        loop_dim(circ_mask, read_file("obj.npy"))
+
+    def test_inpainter(self):
+        lena_image = read_file("lena.npy")
```

Comment:
Please use the `tomopy.misc.phantom.peppers()` or one of the others instead of adding a new test image to the repo.